tests/json/arguments.rs

```diff
@@ -34,4 +37,23 @@ impl Arguments {
 
         Ok(Arguments::InsertOne { document: document })
     }
+
+    pub fn new_insert_many_from_json(object: &Object) -> Result<Arguments, String> {
```

Comment:
I was just using that to signify that they return new instances of the `Arguments` enum. I'll get rid of the prefixes though.